src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorTest.php

```diff
@@ -148,6 +148,8 @@ public function testGetValueNotModifyObjectException()
 
         try {
             $propertyAccessor->getValue($object, 'firstName[1]');
+
+            $this->fail('Expected an exception to be thrown when accessing an invalid element.');
```

Comment:
There's a [separate test case for this already](https://github.com/jakzal/symfony/blob/property-access-test/src/Symfony/Component/PropertyAccess/Tests/PropertyAccessorTest.php#L111-L112). I just realised there's no need to test this again here.